### PR TITLE
ISPN-9598 Near cache simple tutorial

### PR DESCRIPTION
**Pull request by karesti**
https://issues.jboss.org/browse/ISPN-9598

### CODE REVIEW

File: near-cache/src/main/java/org/infinispan/tutorial/simple/nearcache/InfinispanNearCache.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      readCache(nearNumbers);
+
+      // Stop the cache manager and release all resources
+      cacheManager.stop();
+   }
+
+   private static void readCache(RemoteCache<Integer, String> cache) {
+      Instant start = Instant.now();
+      Random random = new Random();
+      random.ints(10_000, 1, 20).forEach(num -> cache.get(num));
```
Comment (diegolovison) [inline near-cache/src/main/java/org/infinispan/tutorial/simple/nearcache/InfinispanNearCache.java]: The values here will be the same.
You need to compare using two get operation.

For the near cache we have
cache.get // remote
cache.get //near

For the NON near cache we have
cache.get // remote
cache.get // remote

**Comment by galderz** [comment]
CI failed because of Spark issue and can't test near cache simple tutorial. @gustavonalle or @karesti , can you implement a workaround?

**Comment by karesti** [comment]
let me try @galderz

**Comment by karesti** [comment]
@gustavonalle @galderz what if we use 9.3.3.Final on spark to avoid using the 9.4.0.Final bom, and when 9.4.1.Final will be out, we delete this line ?

**Comment by gustavonalle** [comment]
@karesti LGTM

**Comment by karesti** [comment]
@galderz should be mergeable now

File: near-cache/src/main/java/org/infinispan/tutorial/simple/nearcache/InfinispanNearCache.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+import org.infinispan.client.hotrod.configuration.NearCacheMode;
+import org.infinispan.client.hotrod.impl.ConfigurationProperties;
+
+public class InfinispanNearCache {
+
+   public static void main(String[] args) {
+      // Create a configuration for a locally-running server
```
Comment (danberindei) [inline near-cache/src/main/java/org/infinispan/tutorial/simple/nearcache/InfinispanNearCache.java]:
```suggestion
      // Create a client configuration connecting to a local server
```

File: near-cache/src/main/java/org/infinispan/tutorial/simple/nearcache/InfinispanNearCache.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      ConfigurationBuilder builder = new ConfigurationBuilder();
+      builder.addServer().host("127.0.0.1").port(ConfigurationProperties.DEFAULT_HOTROD_PORT);
+      builder.nearCache().mode(NearCacheMode.INVALIDATED).maxEntries(20).cacheNamePattern("near-.*");
+
+      // Connect to the server
+      RemoteCacheManager cacheManager = new RemoteCacheManager(builder.build());
+
+      // Create two caches
```
Comment (danberindei) [inline near-cache/src/main/java/org/infinispan/tutorial/simple/nearcache/InfinispanNearCache.java]:
```suggestion
      // Create one remote cache with near caching disabled and one with near caching enabled
```

**Comment by karesti** [comment]
@danberindei I still can't see your suggestion !

**Comment by karesti** [comment]
ok, I just wrote I can't see the suggestion, and the suggestion appeared 🤣

**Comment by karesti** [comment]
@danberindei I added your changes

**Comment by danberindei** [comment]
Integrated, thanks Katia!

**Pull request merged**